docs/src/modules/utils/generateMarkdown.js

```diff
@@ -144,12 +150,22 @@ function generatePropDescription(description, type) {
     }
   }
 
-  return `${deprecated}${jsDocText}${signature}`;
+  let notes = '';
+  if (isElementTypeAcceptingRefProp(type)) {
+    notes +=
+      '<br><sup>*</sup>[Needs to be able to hold a ref](/guides/composition/#Caveat with refs).';
```

Comment:
```suggestion
      '<br>[The component needs to be able to hold a ref](/guides/composition/#Caveat-with-refs).';
```
We already use an asterisk for required props (although the `<sup>` and the description below the table appear to have got lost in the v1 rewrite), but regardless, it's a potential cause for confusion. Since this is directly in the description, no need for the asterisk. It's also appears to be breaking the link:

<img width="1032" alt="image" src="https://user-images.githubusercontent.com/357702/54372422-2e17b600-4673-11e9-934c-69ad4058ca7a.png">

Reply:
Maybe enumerate them instead? Leaves room for additional notes.

Reply:
Yes, now that I look at it again "Type: element type" isn't particularly useful :)

Reply:
I think this is caused by the recent introduction of memo, lazy etc. "element type" is nowadays consistent within the ecosystem. react docs, flow, typescript, prop-types are already using this term. What would you call it?

Reply:
I thought you were suggesting to enumerate the supported element types. If not, then just "element" is fine. "Type" is already covered by the column heading.

Reply:
`element` and `element type` are two different things in React though.

```jsx
const element = <Component /> // <-- element
                 ^^^^^^^^^ element type
// maybe better without jsx
const element = createElement(Component); // <-- element
                              ^^^^^^^^^ element type
```

Reply:
Hmmm, okay. I hadn't considered the distinction.